cutobjects/PbPb/CutsDstoKKpi_PbPb2018_3050_Raa.C

```diff
@@ -196,6 +196,8 @@ AliRDHFCutsDstoKKpi* MakeFileForCutsDs3050_Central2015(bool fUseStrongPID = true
       analysiscuts->SetMaxPtStrongPid(maxPtstrongPID);
     }
     else analysiscuts->SetPidOption(0); //0=kConservative,1=kStrong
+    if(!fIsMC)
+      analysiscuts->EnableNsigmaDataDrivenCorrection(true,AliAODPidHF::kPbPb3050);
 
     analysiscuts->SetOptPileup(false);
     analysiscuts->SetMinCentrality(mincen);

```

Comment:
It does not regard this specific commit, however wouldn't it be better to apply the centrality selection only in data?

Reply:
Done, for all the cuts except those of 2015